File: scalaz-client/src/main/scala/io/swagger/client/api/UserTokenObject.scala
package io.swagger.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import UserTokenObject._

case class UserTokenObject (
  id: String,
username: String,
meta: MetaObject,
sessiontoken: String)

object UserTokenObject {
  import DateTimeCodecs._

  implicit val UserTokenObjectCodecJson: CodecJson[UserTokenObject] = CodecJson.derive[UserTokenObject]
  implicit val UserTokenObjectDecoder: EntityDecoder[UserTokenObject] = jsonOf[UserTokenObject]
  implicit val UserTokenObjectEncoder: EntityEncoder[UserTokenObject] = jsonEncoderOf[UserTokenObject]
}

File: javascript-closure-angular-client/API/Client/UserTokenObject.js
goog.provide('API.Client.userTokenObject');

/**
 * @record
 */
API.Client.UserTokenObject = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.UserTokenObject.prototype.id;

/**
 * @type {!string}
 * @export
 */
API.Client.UserTokenObject.prototype.username;

/**
 * @type {!API.Client._metaObject}
 * @export
 */
API.Client.UserTokenObject.prototype.meta;

/**
 * @type {!string}
 * @export
 */
API.Client.UserTokenObject.prototype.sessiontoken;

